backend/tests/clubs/test_commands.py

```diff
@@ -2,18 +2,71 @@
 import datetime
 import os
 import tempfile
+from unittest import mock
 
+from dateutil.tz import tzutc
 from django.contrib.auth import get_user_model
 from django.core import mail
 from django.core.management import call_command
 from django.core.management.base import CommandError
 from django.test import TestCase
 from django.utils import timezone
+from ics import Calendar
+from ics import Event as Ev
 
 from clubs.models import Club, Event, Favorite, Membership, MembershipInvite, Tag
 from clubs.utils import fuzzy_lookup_club
 
 
+def mocked_requests_get(*args):
+    class MockResponse:
+        def __init__(self, content, status_code):
+            self.text = str(content)
+            self.status_code = status_code
+
+        def text(self):
+            return self.text
+
+    cal = Calendar()
+    event = Ev()
+    event.name = "A test event"
+    event.description = "A test description"
+    event.begin = datetime.datetime(2020, 1, 2, 3, 4, 5)
+    event.end = datetime.datetime(2020, 1, 2, 3, 4, 5) + datetime.timedelta(minutes=60)
```

Comment:
It would be a good idea to make this test use the current time plus a few days. In the future, we may not want to import events that are too far into the past.

In addition, if you use `timezone.now()`, it will add the timezone information so you do not need to use `tzutc`.